RevenueCatUI/Templates/V2/Localizations/LocaleFinder.swift

```diff
@@ -42,6 +42,14 @@ extension Dictionary where Key == String {
             return exactMatch
         }
 
+        // Last resort: matching language and script converting dictionary keys to language and script as well
+        // This covers cases where dictionary keys contain the UN M.49 standard. For example:
+        // If locale = "es_MX" and dictionary key = "es_419" (both have rc_languageAndScript = "es-Latn")
+        if let onlyLanguageAndScriptIdentifier = locale.rc_languageAndScript,
+           let match = self.valueForLanguageAndScript(onlyLanguageAndScriptIdentifier) {
```

Comment:
Not sure if it's possible/worth to add some tests for this?

Reply:
Indeed. Good point that I totally forgot about! Thank you for the suggestion!